apps/charterafrica/payload.config.ts

```diff
@@ -36,13 +36,15 @@ dotenv.config({ path: path.resolve(__dirname, "./.env.local") });
 
 const appURL = process.env.PAYLOAD_PUBLIC_APP_URL;
 
-const cors = process?.env?.PAYLOAD_CORS?.split(",")
-  ?.map((d) => d.trim())
-  ?.filter(Boolean) ?? [];
+const cors =
+  process?.env?.PAYLOAD_CORS?.split(",")
+    ?.map((d) => d.trim())
```

Comment:
Can process be null? I doubt.

Reply:
Yes. This caused the server not to start f null